### PR TITLE
fixed DOSTA formatting error

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Fixing old DOSTA cal csvs that had quotation mark in wrong location - moved to end of source_file name

### CODE REVIEW

File: calibration/DOSTAD/CGINS-DOSTAD-00211__20200203.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-211,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_211_Multipoint_Calibration_2020-02-03.pdf
+211,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_211_Multipoint_Calibration_2020-02-03.pdf"
 211,CC_csv,"[2.77279E-03, 1.20246E-04, 2.18116E-06, 1.69325E+02, -2.23698E-01, -3.75015E+01, 3.33743E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00211__20200203.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00212__20200122.csv [hunk 2]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-212,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_212_Multipoint_Calibration_2020-01-22.pdf
+212,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_212_Multipoint_Calibration_2020-01-22.pdf"
 212,CC_csv,"[2.72677E-03, 1.19463E-04, 2.08215E-06, 1.65933E+02, -2.43220E-01, -3.91162E+01, 3.27282E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00212__20200122.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00375__20200122.csv [hunk 3]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-375,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_375_Multipoint_Calibration_2020-01-22.pdf
+375,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_375_Multipoint_Calibration_2020-01-22.pdf"
 375,CC_csv,"[2.63487E-03, 1.15964E-04, 2.04419E-06, 1.73723E+02, -2.57377E-01, -4.19838E+01, 3.45392E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00375__20200122.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00376__20200122.csv [hunk 4]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-376,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_376_Multipoint_Calibration_2020-01-22.pdf
+376,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_376_Multipoint_Calibration_2020-01-22.pdf"
 376,CC_csv,"[2.69166E-03, 1.16554E-04, 2.20357E-06, 7.61289E+01, -1.04202E-01, -1.87541E+01, 1.50226E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00376__20200122.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00377__20200122.csv [hunk 5]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-377,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_377_Multipoint_Calibration_2020-01-22.pdf
+377,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_377_Multipoint_Calibration_2020-01-22.pdf"
 377,CC_csv,"[2.61058E-03, 1.15567E-04, 2.17695E-06, 1.67230E+02, -2.46162E-01, -4.28557E+01, 3.33326E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00377__20200122.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00378__20200122.csv [hunk 6]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-378,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_378_Multipoint_Calibration_2020-01-22.pdf
+378,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_378_Multipoint_Calibration_2020-01-22.pdf"
 378,CC_csv,"[2.56274E-03, 1.13378E-04, 2.05154E-06, 1.81279E+02, -2.43590E-01, -4.42043E+01, 3.57955E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00378__20200122.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00387__20200116.csv [hunk 7]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-387,CC_conc_coef,"[-7.949502E+00, 9.544519E-01]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_387_Calibration_Certificate_2020-01-16.pdf
+387,CC_conc_coef,"[-7.949502E+00, 9.544519E-01]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_387_Calibration_Certificate_2020-01-16.pdf"
 387,CC_csv,"[2.61152E-03, 1.11088E-04, 2.22897E-06, 2.29540E+02, -3.23989E-01, -5.80968E+01, 4.54244E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00387__20200116.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00388__20200116.csv [hunk 8]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-388,CC_conc_coef,"[-3.315721E+00, 1.056428E+00]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_388_Calibration_Certificate_2020-01-16.pdf
+388,CC_conc_coef,"[-3.315721E+00, 1.056428E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_388_Calibration_Certificate_2020-01-16.pdf"
 388,CC_csv,"[2.81929E-03, 1.20215E-04, 2.42698E-06, 2.29931E+02, -3.17862E-01, -5.84630E+01, 4.54528E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00388__20200116.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00389__20200116.csv [hunk 9]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-389,CC_conc_coef,"[-7.620734E+00, 9.585840E-01]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_389_Calibration_Certificate_2020-01-16.pdf
+389,CC_conc_coef,"[-7.620734E+00, 9.585840E-01]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_389_Calibration_Certificate_2020-01-16.pdf"
 389,CC_csv,"[2.68944E-03, 1.13486E-04, 2.30232E-06, 2.31379E+02, -3.48189E-01, -5.45500E+01, 4.56331E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00389__20200116.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00390__20200116.csv [hunk 10]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-390,CC_conc_coef,"[-9.425620E-01, 9.907772E-01]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_390_Calibration_Certificate_2020-01-16.pdf
+390,CC_conc_coef,"[-9.425620E-01, 9.907772E-01]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_390_Calibration_Certificate_2020-01-16.pdf"
 390,CC_csv,"[2.70195E-03, 1.13673E-04, 2.32805E-06, 2.30788E+02, -3.35029E-01, -5.62763E+01, 4.55250E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00390__20200116.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00391__20200116.csv [hunk 11]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-391,CC_conc_coef,"[3.132769E-01, 1.055467E+00]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_391_Calibration_Certificate_2020-01-16.pdf
+391,CC_conc_coef,"[3.132769E-01, 1.055467E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_391_Calibration_Certificate_2020-01-16.pdf"
 391,CC_csv,"[2.79764E-03, 1.19508E-04, 2.34911E-06, 2.29043E+02, -3.53199E-01, -5.76697E+01, 4.56320E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00391__20200116.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00392__20200116.csv [hunk 12]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-392,CC_conc_coef,"[-1.161001E+00, 1.044199E+00]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_392_Calibration_Certificate_2020-01-16.pdf
+392,CC_conc_coef,"[-1.161001E+00, 1.044199E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_392_Calibration_Certificate_2020-01-16.pdf"
 392,CC_csv,"[2.85167E-03, 1.21360E-04, 2.44218E-06, 2.30473E+02, -3.13485E-01, -5.70360E+01, 4.55563E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00392__20200116.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00503__20200122.csv [hunk 13]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-503,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_503_Multipoint_Calibration_2020-01-22.pdf
+503,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_503_Multipoint_Calibration_2020-01-22.pdf"
 503,CC_csv,"[2.46931E-03, 1.16047E-04, 1.72520E-06, 2.16525E+02, -3.29903E-01, -5.85937E+01, 4.39405E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00503__20200122.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00503__20220617.csv [hunk 14]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-503,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_503_Multipoint_Calibration_2022-06-17.pdf
+503,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_503_Multipoint_Calibration_2022-06-17.pdf"
 503,CC_csv,"[2.49467E-03, 1.09369E-04, 2.04399E-06, 1.94050E+02, -2.56362E-01, -4.99213E+01, 3.93225E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00503__20220617.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00516__20200122.csv [hunk 15]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-516,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_516_Multipoint_Calibration_2020-01-22.pdf
+516,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_516_Multipoint_Calibration_2020-01-22.pdf"
 516,CC_csv,"[2.68171E-03, 1.17343E-04, 2.17994E-06, 1.74229E+02, -2.24825E-01, -4.20225E+01, 3.42855E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00516__20200122.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00517__20200122.csv [hunk 16]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-517,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_517_Multipoint_Calibration_2020-01-22.pdf
+517,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_517_Multipoint_Calibration_2020-01-22.pdf"
 517,CC_csv,"[2.61075E-03, 1.17782E-04, 2.07297E-06, 1.70117E+02, -2.70562E-01, -4.49428E+01, 3.39244E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00517__20200122.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00518__20200122.csv [hunk 17]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-518,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_518_Multipoint_Calibration_2020-01-22.pdf
+518,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_518_Multipoint_Calibration_2020-01-22.pdf"
 518,CC_csv,"[2.71076E-03, 1.17758E-04, 2.23528E-06, 1.93853E+02, -2.67531E-01, -4.84075E+01, 3.81209E+00]",
```
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00518__20200122.csv:3]: looks good

**Review by cdobs — APPROVED** [review]
all look good

File: calibration/DOSTAD/CGINS-DOSTAD-00211__20200203.csv [hunk 18]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-211,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_211_Multipoint_Calibration_2020-02-03.pdf
+211,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_211_Multipoint_Calibration_2020-02-03.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00211__20200203.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00212__20200122.csv [hunk 19]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-212,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_212_Multipoint_Calibration_2020-01-22.pdf
+212,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_212_Multipoint_Calibration_2020-01-22.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00212__20200122.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00375__20200122.csv [hunk 20]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-375,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_375_Multipoint_Calibration_2020-01-22.pdf
+375,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_375_Multipoint_Calibration_2020-01-22.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00375__20200122.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00376__20200122.csv [hunk 21]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-376,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_376_Multipoint_Calibration_2020-01-22.pdf
+376,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_376_Multipoint_Calibration_2020-01-22.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00376__20200122.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00377__20200122.csv [hunk 22]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-377,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_377_Multipoint_Calibration_2020-01-22.pdf
+377,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_377_Multipoint_Calibration_2020-01-22.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00377__20200122.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00378__20200122.csv [hunk 23]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-378,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_378_Multipoint_Calibration_2020-01-22.pdf
+378,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_378_Multipoint_Calibration_2020-01-22.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00378__20200122.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00387__20200116.csv [hunk 24]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-387,CC_conc_coef,"[-7.949502E+00, 9.544519E-01]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_387_Calibration_Certificate_2020-01-16.pdf
+387,CC_conc_coef,"[-7.949502E+00, 9.544519E-01]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_387_Calibration_Certificate_2020-01-16.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00387__20200116.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00388__20200116.csv [hunk 25]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-388,CC_conc_coef,"[-3.315721E+00, 1.056428E+00]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_388_Calibration_Certificate_2020-01-16.pdf
+388,CC_conc_coef,"[-3.315721E+00, 1.056428E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_388_Calibration_Certificate_2020-01-16.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00388__20200116.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00389__20200116.csv [hunk 26]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-389,CC_conc_coef,"[-7.620734E+00, 9.585840E-01]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_389_Calibration_Certificate_2020-01-16.pdf
+389,CC_conc_coef,"[-7.620734E+00, 9.585840E-01]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_389_Calibration_Certificate_2020-01-16.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00389__20200116.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00390__20200116.csv [hunk 27]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-390,CC_conc_coef,"[-9.425620E-01, 9.907772E-01]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_390_Calibration_Certificate_2020-01-16.pdf
+390,CC_conc_coef,"[-9.425620E-01, 9.907772E-01]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_390_Calibration_Certificate_2020-01-16.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00390__20200116.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00391__20200116.csv [hunk 28]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-391,CC_conc_coef,"[3.132769E-01, 1.055467E+00]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_391_Calibration_Certificate_2020-01-16.pdf
+391,CC_conc_coef,"[3.132769E-01, 1.055467E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_391_Calibration_Certificate_2020-01-16.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00391__20200116.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00392__20200116.csv [hunk 29]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-392,CC_conc_coef,"[-1.161001E+00, 1.044199E+00]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_392_Calibration_Certificate_2020-01-16.pdf
+392,CC_conc_coef,"[-1.161001E+00, 1.044199E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_392_Calibration_Certificate_2020-01-16.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00392__20200116.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00503__20200122.csv [hunk 30]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-503,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_503_Multipoint_Calibration_2020-01-22.pdf
+503,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_503_Multipoint_Calibration_2020-01-22.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00503__20200122.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00503__20220617.csv [hunk 31]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-503,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_503_Multipoint_Calibration_2022-06-17.pdf
+503,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_503_Multipoint_Calibration_2022-06-17.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00503__20220617.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00516__20200122.csv [hunk 32]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-516,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_516_Multipoint_Calibration_2020-01-22.pdf
+516,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_516_Multipoint_Calibration_2020-01-22.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00516__20200122.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00517__20200122.csv [hunk 33]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-517,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_517_Multipoint_Calibration_2020-01-22.pdf
+517,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_517_Multipoint_Calibration_2020-01-22.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00517__20200122.csv:2]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00518__20200122.csv [hunk 34]
```diff
@@ -1,3 +1,3 @@
 serial,name,value,notes
-518,CC_conc_coef,"[0.0, 1.0]","[offset, slope]"; source_file DOSTA-D_Optode-4831_SN_518_Multipoint_Calibration_2020-01-22.pdf
+518,CC_conc_coef,"[0.0, 1.0]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_518_Multipoint_Calibration_2020-01-22.pdf"
```
Comment (cooleyky) [inline calibration/DOSTAD/CGINS-DOSTAD-00518__20200122.csv:2]: looks good

**Review by cooleyky — APPROVED** [review]
all corrections look good!

**Pull request merged**